src/App.tsx

```diff
@@ -1,223 +1,192 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Condition } from './types/Condition';
+import { Todo } from './types/Todo';
+import { addTodos, deleteTodos, getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState('');
```

Comment:
It's not a boolean value, don't confuse other developers
```suggestion
  const [errorText, setErrorText] = useState('');
```